vector/src/main/res/layout/dialog_ssl_fingerprint.xml

```diff
@@ -1,53 +1,45 @@
 <?xml version="1.0" encoding="utf-8"?>
-
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-              android:orientation="vertical"
-              android:layout_width="match_parent"
-              android:layout_height="match_parent">
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
```

Comment:
no padding for this one?

Reply:
Didn't have a way to view this dialog, I can add padding but I won't be able to test.